sponsoring.md

```diff
@@ -0,0 +1,116 @@
+| Superduper Company | Edith Clarke | `edith@super.com` |	24.1.2017 |	Tina | Cancelled | Ask them next year again | - | - | - | - |
+| Amazing Corporation | Ada Lovelace | `ada@amazing.com` |	24.1.2017 |	Tina | Sent |  | - | - | - | - |
+| New Startup | Radia Perlman | `radia@startup.com` |	27.1.2017 |	Tina | Confirmed - Silver | Remind them of sponsoring material | 10.2.2017 | yes | - | 10.2.2017 |
+
+We also use `Conditional Formatting` to count the number of sponsors per level and to highlight the status from the requests.
+
+In another tab we calculate the money we get from the sponsors to have a little overview how much money we already collected.
+
+![Sponsor overview](material/sponsor-overview.png)
+
+### Sponsoring plans
+
+#### What do the sponsors get
+
+Depending on the sponsoring level the sponsors get from us different things.
+
+On the one hand they get free tickets and options to buy more tickets (in case we are sold out these tickets are still buyable for the sponsors).
+
+On the other hand they get a lot of opportunities to advertise their company. Depending on the level they may put up posters or booths to talk to attendees. We also mention some in tweets or in any mails we send to our participants.
+
+#### What do the sponsors provide
+
+The most trivial thing to ask sponsors for is money. As you will read we offered several sponsoring levels to choose from.
+
+Some sponsors ask to provide something special to separate them from others.
+Some ideas for these specials are support for:
+ - childcare
+ - coffee
+ - party
+
+And also here you can differentiate between organizing this yourself and hand over the invoice or you trust the sponsor to completely handle this by themselves. The advantage of the latter is less organizational effort for you :). But you have to trust them that they really take care of this.
+
+### Find sponsors
+
+Don't be scared but finding sponsors is an exhausting task with a lot of communication. It is also a very important and refreshing tasks if you like to organize and talk to people.
+
+#### Use your network
+
+At first: Use your own network to find sponsors. Talk to people in your company if they want to sponsor your event. Then ask friends. If you are a member of a meet-up or community use these connections to advertise your event.
+
+If you already ran a conference ask the sponsors from last year again.
+
+#### Use everything else
+
+It is better to have too much sponsors then too little. If you don't know whom to ask anymore have a look at other conferences or meet-ups near you to have a look at their sponsoring list.
+
+Another thing is to ask for sponsoring or support on social media platforms like twitter to support your event.
+
+------------------------
+### Sample letter
+
+This is our latest cover letter for JSUnconf 2017. We sent this along with a set of slides to all our sponsoring requests.
```

Comment:
i am impressed how far our first very basic letters have developed. It also makes me fear that it looks too daunting for people that start just right now and have nothing right now. maybe we can add something like:


**Important!** This is our 4th iteration after 4 years. We started with a much more basic design, mainly text. We have the luck of having a designer in the team, so don't be daunted. In case you are curious, here is our first version: [link]

---

I can try to find our first sponsoring material and we can add it too, so people see that its possible to get a lot of sponsors also when they have no design skills.

Yay/Nay?
I could send a follow up PR after merge?  Yay/Nay?